sdk/netapp/Microsoft.Azure.Management.NetApp/src/Properties/AssemblyInfo.cs

```diff
@@ -9,7 +9,7 @@
 [assembly: AssemblyDescription("Provides management functionality for Azure NetApp Storage.")]
 
 [assembly: AssemblyVersion("1.0.0.0")]
```

Comment:
Need to update `AssemblyVersion` to be consistent with the version number (e.g. `1.16.0.0`). Actually `AssemblyVersion` is used by runtime. See: https://docs.microsoft.com/en-us/troubleshoot/developer/visualstudio/general/general/assembly-version-assembly-file-version

Reply:
version bumped, we have been updating `AssemblyFileVersion` only will update `AssemblyVersion` from now on, thanks